src/components/CardContainer/api/constants/endpoints.ts

```diff
@@ -0,0 +1,4 @@
+export const enum Endpoints {
+    PHOTO_BASE_ENDPOINT = "https://www.themoviedb.org/t/p/w220_and_h330_face",
+    POPULAR_MOVIE_BASE_EP = "https://api.themoviedb.org/3/movie/popular?api_key=",
```

Comment:
your enum is named Endpoints, so no need to add `BASE_EP` etc.

Reply:
done